src/components/BookList/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { useQuery } from 'react-query';
+
+import { getBooks } from '../../services/BooksService';
+import Loading from '../Spinner/components/loading';
+import { IBook } from '../../utils/types';
+
+import styles from './styles.module.scss';
+
+function NavBar() {
+  const { data, isLoading } = useQuery('book', getBooks);
+  const books = data?.page || [];
+  return (
+    <div className={styles.container}>
+      {isLoading ? (
+        <Loading />
+      ) : (
+        books.map((todo: IBook) => (
```

Comment:
```suggestion
        books.map((book: IBook) => (
```
Use more descriptive names